.github/workflows/pages.yml

```diff
@@ -0,0 +1,31 @@
+name: "CI Docs"
+
+on:
+  push:
+    branches:
+    - master
```

Comment:
The default branch of this repo is called ``main`` instead of ``master``.

Replace ``- master`` by     ``- main``.